test/ctia/http/routes/common_test.clj

```diff
@@ -2,20 +2,24 @@
   (:require [ctia.http.routes.common :as sut]
             [clj-momo.lib.clj-time.coerce :as tc]
             [clj-momo.lib.clj-time.core :as t]
-            [clojure.test :refer [is deftest]]))
+            [clojure.test :refer [is deftest testing]]))
 
-(deftest coerce-from
+(deftest coerce-date-range
   (with-redefs [sut/now (constantly (tc/from-string "2020-12-31"))]
     (let [from (tc/from-string "2020-04-01")
           to (tc/from-string "2020-06-01")]
-      (is (= (tc/from-string "2019-12-31")
-             (sut/coerce-from (tc/from-string "2019-12-30") nil)))
-      (is (= (tc/from-string "2019-06-01")
-             (sut/coerce-from (tc/from-string "2019-06-1") to)))
-      (is (= from
-             (sut/coerce-from from nil)))
-      (is (= from
-             (sut/coerce-from from to))))))
+      (is (= {:gte (tc/from-string "2019-12-31")
+              :lt (sut/now)}
```

Comment:
Can't comparing two `now`s (expected/actual) result in flaky tests?

```clj
(= (java.util.Date.)
   (do
     (Thread/sleep 5)
     (java.util.Date.))) ;; => false
```

Reply:
There is a `with-redefs` of `now` that wraps that execution to ensure that this function always returns the same date.

Reply:
👍didn't realise!